Libraries/CommonFunctions.psm1

```diff
@@ -722,7 +722,11 @@ function Install-CustomLIS ($CustomLIS, $customLISBranch, $allVMData, [switch]$R
 						Write-LogInfo "Old LIS: $currentlisVersion"
 						Write-LogInfo "New LIS: $upgradedlisVersion"
 						Add-Content -Value "Old LIS: $currentlisVersion" -Path ".\Report\AdditionalInfo-$TestID.html" -Force
-						Add-Content -Value "New LIS: $upgradedlisVersion" -Path ".\Report\AdditionalInfo-$TestID.html" -Force
+                        Add-Content -Value "New LIS: $upgradedlisVersion" -Path ".\Report\AdditionalInfo-$TestID.html" -Force
+                        if ($upgradedlisVersion -eq $currentlisVersion) {
+                            Write-LogErr "LIS Version is not changed."
```

Comment:
In the overall test result is this mesage clear enough to understand it's in the upgrade scenario?
Otherwise, change the text maybe, like:
Write-LogErr "LIS Version is not changed after LIS upgrade."

Reply:
@chvalean  Message updated to - 
`LIS Version is not changed even after successful LIS RPMs installation.`

Reply:
> @chvalean Message updated to -
> `LIS Version is not changed even after successful LIS RPMs installation.`

Please add PR verified and post some test results.
Feel free to merge the PR then, approved it now but unsure to merge without that.

Reply:
@chvalean Sure. will merge once verified.